test/new-e2e/tests/installer/upgrade_scenario_test.go

```diff
@@ -175,6 +175,28 @@ func (s *upgradeScenarioSuite) TestStopWithoutExperiment() {
 	require.Equal(s.T(), beforeStatus.Packages["datadog-agent"], afterStatus.Packages["datadog-agent"])
 }
 
+func (s *upgradeScenarioSuite) TestPromoteWithoutExperiment() {
+	s.RunInstallScript("DD_REMOTE_UPDATES=true")
+	defer s.Purge()
+	s.host.WaitForUnitActive(
+		"datadog-agent.service",
+		"datadog-agent-trace.service",
+		"datadog-agent-process.service",
+		"datadog-installer.service",
+	)
+
+	s.setCatalog(testCatalog)
+
+	beforeStatus := s.getInstallerStatus()
+
+	_, err := s.promoteExperimentCommand()
+	require.Error(s.T(), err)
+
+	afterStatus := s.getInstallerStatus()
+	require.Equal(s.T(), beforeStatus.Packages["datadog-agent"], afterStatus.Packages["datadog-agent"])
+	require.Equal(s.T(), beforeStatus.Version, afterStatus.Version)
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Let's add a `start experiment` + `promote experiment` after that, to make sure the installer is not in a fully broken state and can recover